### PR TITLE
Add support for monitoring ingress events.

### PR DESCRIPTION
**Pull request by nabadger**
Added support for monitoring ingress events.

### CODE REVIEW

**Comment by nabadger** [comment]
Looks like the merge conflict is because there's no a `switch` statement in use.

Is this something you want me to fix on my branch? (can do)

**Comment by nabadger** [comment]
I think this is good now - let me know if there's issues - thanks again.

File: pkg/event/event.go [hunk 1]
```diff
@@ -75,6 +77,8 @@ func New(obj interface{}, action string) Event {
 		component = string(object.Spec.Type)
 	case *api_v1.Secret:
 		kind = "secret"
+		reason = action
```
Comment (tylerauerbeck) [inline pkg/event/event.go]: @nabadger Is there a reason you're adding `reason` and `action` back into the case for Secret? These are set above, so this is unnecessary. Was there something that you saw that required it?

Comment (nabadger) [reply pkg/event/event.go]: @tylerauerbeck No sorry - it looks like my merge from upstream to resolve conflicts didn't remove those lines. 

I've fixed it now - thanks.

**Comment by tylerauerbeck** [comment]
LGTM

**Comment by Mumrau** [comment]
I do belive that the RBAC has been forgotten in that PR, my kubewatch doesn't have the rights to watch the ingresses, even when created with rbac.create=true.
I don't know if it's a good idea to discuss this here, I can create an issue if needed.

**Pull request merged**